commercetools/provider.go

```diff
@@ -54,13 +70,13 @@ func providerConfigure(d *schema.ResourceData) (interface{}, error) {
 		ClientID:     d.Get("client_id").(string),
 		ClientSecret: d.Get("client_secret").(string),
 		Scopes:       []string{fmt.Sprintf("manage_project:%s", projectKey)},
```

Comment:
Maybe this can be done in another PR, but can we read the scopes from the environment variable `CTP_SCOPES`?